<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="studentDisplay">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>690</width>
     <height>511</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>229</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>630</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Gradebook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveStudentInfoButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>211</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Student Information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
